docs/pact-reference.md

```diff
@@ -649,8 +649,6 @@ Module-global constant values can be declared with [defconst](#defconst).
 
 Pact code can be explicitly typed, and is always strongly-typed under the hood as the native
 functions perform strict type checking as indicated in their documented type signatures.
-language, but does use fixed type representations "under the hood"
-and does no coercion of types, so is strongly-typed nonetheless.
```

Comment:
Seemed like a stub from an old revision, and didn't appear to add new information.